src/gf_nf4.h

```diff
@@ -95,6 +95,7 @@ class NF4 : public gf::Field<T> {
 template <typename T>
 NF4<T>::NF4(unsigned n) : gf::Field<T>(T(65537), n)
 {
+    this->isNF4 = true;
```

Comment:
Why do we need this?

Reply:
Ok, I saw it.
I think it can be improved, but that's OK for now: you can ignore this comment.